File: Pythonleetcode/Leetcode1.py
# # Approach 1
# class Solution(object):
#     def twoSum(self,nums, target):
#         for i, num in enumerate(nums):
#             sub_num = target - num
#             if sub_num in nums:
#                 index1 = nums.index(sub_num)
#                 if index1 != i:
#                     return [i, index1]

# Approach 2
class Solution(object):
    def twoSum(self,nums, target):
        for i, num in enumerate(nums):
            sub_num = target - num
            if sub_num in nums:
                index1 = nums.index(sub_num)
                if index1 != i:
                    return [i, index1]

print(Solution().twoSum([2,0,17, 7, 11, 15],9))

File: CppPractice/Project1/Project1/Leetcode1.cpp
#include "LeetcodeHeader.h"

class Solution {
public:
    vector<int> twoSum(vector<int>& nums, int target) {
        vector<int> ans(2, 0);
        for (int i = 0; i < nums.size() - 1; i++)
        {
            ans = { i,0 };
            for (int j = i + 1; j < nums.size(); j++)
            {
                if (nums[j] == target - nums[i])
                {
                    ans[1] = j;
                    return ans;
                }
            }

        }


        return ans;
    }
};